src/prog/main.F90

```diff
@@ -958,6 +958,9 @@ subroutine xtbMain(env, argParser)
    type is(TxTBCalculator)
       call write_energy(env%unit,res,fres, &
         & (set%runtyp.eq.p_run_hess).or.(set%runtyp.eq.p_run_ohess).or.(set%runtyp.eq.p_run_bhess))
+   type is(TOniomCalculator)
+      call write_energy_oniom(env%unit,res,fres, &
+         & (set%runtyp.eq.p_run_hess).or.(set%runtyp.eq.p_run_ohess.or.(set%runtyp.eq.p_run_bhess)))
```

Comment:
This has no effect on functionality of course, but is there a specific reason why you added the brackets like that?

Reply:
nah, just a typo